File: frontend/src/models/Article.ts
class Article {
    title: string;
    text: string;
    authorId: number;
    categoryId: number;
    tags: number[];

    constructor(title: string, text: string, authorId: number, categoryId: number, tags: number[]) {
        this.title = title;
        this.text = text;
        this.authorId = authorId;
        this.categoryId = categoryId;
        this.tags = tags;
    }

}

export { Article }

File: backend/News.Database/Entities/Article.cs
using News.Database.Common;

namespace News.Database.Entities
{
    public class Article : BaseEntity
    {
        public int Id { get; set; }

        public string Title { get; set; }

        public string Text { get; set; }

        public int AuthorId { get; set; }

        public int CategoryId { get; set; }

        public virtual Author Author { get; set; }

        public virtual Category Category { get; set; }

        public virtual ICollection<Tag> Tags { get; set; }
    }
}
